bb/ramfs.go

```diff
@@ -114,9 +114,10 @@ func ramfs() {
 	}
 
 	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = []string{"init", "ubin"}
+		
 
 	if *extraPaths != "" {
-		extras := strings.Split(*extraPaths, " ")
+		extras := strings.Split(strings.Trim(*extraPaths, " "), " ")
```

Comment:
Mind using strings.TrimSpace instead? It'll trim all kinds of whitespace characters, not just the regular ascii space.